presto-docs/src/main/sphinx/security/group-file.rst

```diff
@@ -0,0 +1,42 @@
+==========
+Group File
+==========
+
+Presto can map user names onto groups for easier access control and
+resource group management. Group file resolves group membership using
+a file on the coordinator.
+
+Group File Configuration
+------------------------
+
+Enable group file by creating an ``etc/group-provider.properties``
+file on the coordinator:
+
+.. code-block:: none
+
+    group-provider.name=file
+    file.group-file=/path/to/group.txt
```

Comment:
txt? We usually use json. I think using json make sense here because:
 - it is coherent with other file based configuration files
 - it is much easier to extend it in future
 - there is less parsing code

Reply:
This is based on the Apache HTTP server format, which is the format we use for password files

Reply:
We used the Apache password file format since it is a standard format and we could have people use `htpasswd`, rather than needing to write our own or have detailed instructions on how to create it. I don't think this applies here.

However, I can see the argument that this is a very simple mapping format that we do not expect to change, and using JSON is overkill.